server_admin/topics/login-settings/forgot-password.adoc

```diff
@@ -1,18 +1,18 @@
 
 == Enabling forgot password
 
-If you enable `Forgot Password`, users can reset their login credentials if they forget their passwords or lose their OTP generator. 
+If you enable `Forgot password`, users can reset their login credentials if they forget their passwords or lose their OTP generator. 
 
 .Procedure
-. Click *Realm Settings* in the menu.
+. Click *Realm settings* in the menu.
 . Click the *Login* tab.
 +
 .Login tab
 image:{project_images}/login-tab.png[Login Tab]
 +
-. Toggle *Forgot Password* to *ON*.
+. Toggle *Forgot password* to *On*.
```

Comment:
```suggestion
. Toggle *Forgot password* to *ON*.
```

Reply:
Here's the style guideline related to toggles; https://docs.google.com/document/d/1YBu8uTkSatmYA4rdjymSeWBPl_LMZ5igM0NSssldz00/edit#heading=h.fuzmtiq14qng